### ISSUE TITLE
Delete twisted.flow

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3726]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3726|
|Type|enhancement|
|Created|2009-03-30 23:12:22Z|
|Branch|https://github.com/twisted/twisted/tree/remove-flow-3726|

It has been deprecated for long enough.

<details><summary>Searchable metadata</summary>

```
trac-id__3726 3726
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_remove_flow_3726 branches/remove-flow-3726
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__flow flow
keywords__ 
time__1238454742000000 1238454742000000
changetime__1239710594000000 1239710594000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [26673]) Branching to 'remove-flow-3726'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Okay.  It's deleted in the branch.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

YES! Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [26699]) Merge remove-flow-3726

Author: exarkun
Reviewer: therve
Fixes: [#3726](https://github.com/twisted/twisted/issues/3726)

Remove Twisted Flow, long unmaintained and deprecated.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
